README.md

```diff
@@ -18,7 +18,7 @@ Note that we have only tested it on [AWS CloudHSM](https://aws.amazon.com/cloudh
 
 Please note that this software does not provide any authentication or authorization. You will need to take care of that yourself. It simply returns the signature for valid payloads, after performing some checks:
 * Is the message a valid payload?
-* Does the message begin with a 0x01 or 0x02? Indicating it is a baking or endorsement, rather than a 0x03 transfer.
+* Does the message begin with a 0x11, 0x12 or 0x13? Indicating it is a baking or endorsement, rather than a 0x03 transfer.
```

Comment:
0x03 could also be votes now and not just transfers. The signer checks for that: https://github.com/oxheadalpha/tezos-kms-signer/blob/926601fb0593229182238aba952abb57a4fb1a44/src/sigreq.py#L36-L53

Also maybe add 
```suggestion
* Does the message begin with a 0x11, 0x12 or 0x13? Indicating it is a baking, preendorsement, or endorsement, rather than a 0x03 transfer.
```

Reply:
actually we should forbid any 0x03 operation. With a consensus key baker, you vote with the baker key, not the consensus key.

Reply:
That's assuming all users would use a consensus key. But i don't think we need to restrict them to that. As far as i can tell, there is no issue with allowing votes. If we keep the voting we should add in that voting apps are accepted.